### ISSUE TITLE
Windowed Modal not reaching bottom of screen

### ISSUE THREAD

**Issue by flodaniel** [mukaschultze/nativescript-windowed-modal#15]
### Which platform(s) does your issue occur on?
- Android 

### Please, provide the following version numbers that your issue occurs with:

- CLI: 6.1
- Cross-platform modules: 6.1.
- Runtime(s): 6.1 & 6.1
- Angular with Typescript

### Please, tell us how to recreate the issue in as much detail as possible. 
Create a non fullscreen modal with the following code. Use verticalPosition="bottom". The modal will not stick to the very bottom of the screen. There will be some space in between.
I tried with the current version and with @next  but the result is the same. 

I am unable to use ExtendedShowModalOptions, as this will lead to some errors. I registered the plugin in main.ts with the following code: 

```
import { ExtendedShowModalOptions, ModalStack, overrideModalViewMethod } from "nativescript-windowed-modal"
import { registerElement } from "nativescript-angular/element-registry";

overrideModalViewMethod()
registerElement("ModalStack", () => ModalStack as any)
```

And here is the code to reproduce the issuse: 

```
  const options: any = {
            viewContainerRef: viewContainerRef,
            fullscreen: fullscreen,
            context: context,
            animated: true,
            dimAmount: 0.5
        };

        return this.modalService.showModal(type, options).then(dialogResult => {
            this.modalOpen = false;
            return dialogResult;
        });
```
```
  <ModalStack verticalPosition="bottom">
      <GridLayout backgroundColor="#fff">
          <Label text=" laskdjföalsdf"></Label>
      </GridLayout>
  </ModalStack>
```

Here is a screenshot with layout markes enabled:
![image](https://user-images.githubusercontent.com/6661557/67944122-05c9d100-fbdc-11e9-8925-83d8b03f4826.png)

**Comment by flodaniel** [comment]
hi @mukaschultze ,
do you know if you can fix this? are there plans to fix this?
I would really need it 👍

**Comment by mukaschultze** [comment]
No ETA at the moment, PRs are welcome in case anyone knows how to fix it.

**Comment by mukaschultze** [comment]
> I am unable to use ExtendedShowModalOptions, as this will lead to some errors. I registered the plugin in main.ts with the following code:

What errors occur? [This file](https://github.com/mukaschultze/nativescript-windowed-modal/blob/master/demo-ng/src/app/app.component.ts) has some working TS examples of how to use `ExtendedShowModalOptions`.

Also, [nativescript-material-bottomsheet](https://market.nativescript.org/plugins/nativescript-material-bottomsheet) might be a better alternative to a bottom aligned modal.

**Comment by mukaschultze** [comment]
@Firetrip I couldn't reproduce the issue, the modal sticks to the bottom as it should both on a real device and in the emulator. In what devices you observed that behavior?

**Comment by flodaniel** [comment]
It is a Xiaomi Mi 9. Maybe it is because of the cuved bottom of the screen

**Comment by flodaniel** [comment]
Indeed, I just tried on an older phone with software buttons, which the Mi 9 does not have (it uses gestures) and the modal reaches the bottom

**Comment by Tyler-V** [comment]
I found this occuring when the modal was not set to fullscreen, I put up a PR that resolves this for Android.

**Comment by blasus** [comment]
@Firetrip I found a workaround for this issue using the combination of _fullscreen: false_ and _stretched: false_ in the ExtendedShowModalOptions object. Let me know if it works to you as well
